openreview/conference/templates/programchairWebfield.js

```diff
@@ -959,31 +959,53 @@ var displayStatsAndConfiguration = function(conferenceStats) {
 
   // Timeline
   var invitationMap = conferenceStatusData.invitationMap;
-
   html += '<div class="col-md-8 col-xs-12">'
   html += '<h4>Timeline:</h4><ul style="padding-left: 15px">';
-  html += renderInvitation(invitationMap, SUBMISSION_ID, 'Paper Submissions')
-  html += renderInvitation(invitationMap, REVIEWERS_ID + '/-/' + BID_NAME, 'Reviewers Bidding')
+  var not_dated_elements = []
+  var dated_elements = []
+  var push_to_dated_arrays = function(invitationMap, id, name, dated, not_dated){
+    var curr_content = renderInvitation(invitationMap, id, name)
+    if (curr_content)
```

Comment:
It's better to return early: `if (!currContent) return;`. It keeps things easier to read and prevents conditions from nesting too deeply.

Reply:
Yup, I've made those changes